specifyweb/frontend/js_src/lib/components/Merging/CompareSubView.tsx

```diff
@@ -147,6 +147,10 @@ function MergeDialog({
     ...[mergedRecords, ...children].map((children) => children.length)
   );
 
+  const childrenLength = resources
+    .map((record) => f.maybe(record, getChildren))
+    .filter((children) => children && children.length > 0).length;
```

Comment:
Instead of doing the implicit conversion to a boolean here, can we explicitly check for undefined? 
Something like `typeof children !== undefined && children.length > 0`

Reply:
also, you don't care about the number of children - you care if there is more than one column that has at least 1 child - i.e if the first column has two children, the checkbox should still not be displayed, it's only if there are children in more than column that there is something to compare to, and thus the "show conflicting fields only" checkbox makes sense.

Reply:
children is not undefined when 0 records. Agree I need to change to > 1 not > 0